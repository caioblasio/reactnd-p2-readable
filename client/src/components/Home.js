import React from 'react'
import PostList from './PostList'

const Home = () => {
    return <PostList />
}

export default Home