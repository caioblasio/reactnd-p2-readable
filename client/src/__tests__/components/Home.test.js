import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Home from '../../components/Home';


describe('<Home />', () => {

    test('renders the component', () => {
      const wrapper = shallow(<Home />);
      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });

});