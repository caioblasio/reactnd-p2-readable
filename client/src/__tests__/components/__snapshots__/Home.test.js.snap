// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders the component 1`] = `
<Fragment>
  <WithStyles(Connect(PostList))
    headline="All posts"
  />
  <Link
    replace={false}
    to="/new"
  >
    <WithStyles(Button)
      aria-label="Add"
      className="Home-button-1"
      color="primary"
      variant="fab"
    >
      <pure(AddIcon) />
    </WithStyles(Button)>
  </Link>
</Fragment>
`;
